### DISCUSSION TITLE
推薦的輕小說

### DISCUSSION THREAD

**Discussion by Cerosop** [1101-GS4719-C/Python-Community-GS4719-C#1815] [General]
我自己蠻喜歡看輕小說的，但最近不知道該看些什麼，不知道大家有什麼推薦的嗎?

我自己非常推薦青春豬頭少年跟在地下城尋求邂逅是否搞錯了甚麼(沒錯名字就是這麼長)。前者是在說青少年可能會遇到的種種內心的問題以及作者建議應該保有的心態，是一部可以學到很多東西也很讓人感動的作品。後者則是在講述一個成長比常人快很多的冒險故事，聽起來跟其他大眾作品差不多但這部不管是世界觀還是文筆都十分厲害，上面兩部我都看了2次以上了，強力推薦。

**Comment by alvin119** [comment]
我推薦一本很有名的輕小說叫作《果然我的青春戀愛喜劇搞錯了。》，簡稱《果青》。
《果青》講述了青少年們的成長，對於校園生活裡對於人際關係、自我認同的描寫都很寫實，
作者文筆我也覺得還不錯，十分推薦~

**Comment by B1nn9** [comment]
我推薦《破滅的倒懸者》。這本書在寫看似一名普通大學生的主角因為意外目擊一場怪異事件，而被捲入國家機密相關案件中，導致他必須協助特務搜查員共同調查，以找出犯人並追回檔案。作者對於角色塑造、心理戰術描寫得十分細膩，不過讀者需要在尋求真相的過程中多一點耐心才行。

**Comment by B1nn9** [comment]
我推薦《奇妙故事集 墓碑鎮》。這本書作者是伊藤潤二，書中擷取他數部經典漫畫，並由曾配合過的動畫編劇改寫成文字。故事充滿不少弔詭的劇情，雖然有些駭人聽聞卻又讓我深深入迷，所以即便是曾看過原漫畫的人，一定也能感受到新意。

**Comment by B1nn9** [comment]
我推薦《偵探已經，死了。》。這本書實際上奇幻的成分多過於懸疑，劇情鋪陳也並非以解謎為主軸發展，而是充滿了更刺激的冒險情節，故事內容則是由奇幻、戀愛等元素交織而成，特別的是故事整體採倒敘法呈現，儘管一開始就讓讀者知道角色之一已死，卻絲毫不減劇情的精彩度。

**Comment by B1nn9** [comment]
我推薦《小書痴的下剋上：為了成為圖書管理員不擇手段！》。書中主角為了在轉生後的新世界中繼續看書，不斷透過各種方法土法煉鋼，試圖讓書本能被發明。整體故事圍繞在「創造」的主題上，雖然概念看似簡單且進展細膩略為緩慢，但作者想傳遞的精神卻是能讓人一再回味的。

**Comment by B1nn9** [comment]
我推薦《龍與雀斑公主》。這本書以「活出另一個世界的你」作為主軸，講述一個愛唱歌的高中女生因故無法再開口，直到她和另一個現實世界相遇，並在那大放異彩。內容帶有穿越情節且富含奇幻色彩，加上由於同名電影已問世，讓人能同時體驗到小說的細膩和電影的聲光效果。

**Comment by B1nn9** [comment]
我推薦《關於我轉生變成史萊姆這檔事》。這本書以一個平凡上班族作為故事主角，而在他被隨機殺人犯殺死之後竟轉生變成了被視為能力最弱的雜魚魔物「史萊姆」，但主角卻在誤打誤撞之下逐漸成為魔物國的領袖，整個過程處處充滿驚奇，是一本容易讓喜愛新奇風格的讀者產生興趣的奇幻之作。

**Comment by B1nn9** [comment]
我推薦《果然我的青春戀愛喜劇搞錯了》。這本書以一個彆扭的高中男生與校花之間的相遇出發，當中不只有戀愛故事的浪漫設定，還充滿了青少年自我認同的探討，是一部相當細膩、用心的作品。雖然故事的角色設定較不真實，並非所有人都能有所共鳴，但仍值得一讀。若有興趣深入的話，另外則還有其它番外篇。

**Comment by B1nn9** [comment]
我推薦《雨洗過的斜陽》。這本書在千篇一律的校園愛情故事中加入了些許懸疑、靈異成分，令人對整體情節感到耳目一新。故事以告別與成長作為核心，除了愛情外也有針對學生時代的友誼做描述，作者筆觸柔軟總能勾起讀者的青春回憶，輕而不淡的遺憾感慨讓人難以忘懷。

**Comment by B1nn9** [comment]
我推薦《關於我在無意間被隔壁的天使變成廢柴這件事》。這本書講述了高中生藤宮周和他的鄰居椎名真晝之間的戀愛喜劇。周是個不擅長社交的自閉少年，真晝是個被稱為「天使」的美少女，但實際上她有著不為人知的一面。兩人因為一把雨傘而相識，並逐漸發展出一段超越友情的關係。故事的主要看點是周和真晝的甜蜜互動，以及他們如何克服各自的心理創傷，互相扶持，成長為更好的人。

**Comment by B1nn9** [comment]
我推薦《藥師少女的獨語》。這本書講述了一位擁有精明推理能力和豐富藥理知識的少女，原本在花街出身並擔任藥師，但被人販子綁架後賣到後宮擔任下級的女官。在勾心鬥角的宮廷中，她憑藉自身的學識和技能，排憂解難、逐漸建立人脈，也發掘出宮廷裡不為人知的一面。同時，她也與管理後宮的宦官展開了一段不尋常的戀情。故事的主要看點是主角的藥學和推理才能，以及她與宦官的甜蜜互動。讀者可以欣賞到主角如何運用藥物和智慧解決各種宮廷的難題，也可以感受到其中的浪漫與幽默。

**Comment by B1nn9** [comment]
我推薦《歡迎來到實力至上主義的教室 2年級篇》。這本書講述了成功在高中度過一年級的主角將在二年級時面對來路不明的一年級新生，同時與來自白房的刺客進行攻防戰。在這所學校二年級將是決定命運的一年，一切皆由自身實力來衡量，是一間完全的實力主義至上學校。主角必須利用自己的智慧和技能，以及與同班同學的合作，來應對各種特別考試和挑戰，同時也揭露了自己的過去和真實目的。故事的主要看點是主角所表現出的冷靜分析以及各種策略，讓讀者可以欣賞到他如何運用實力至上主義的教室的規則和漏洞，來翻盤和勝利，也可以感受到他與其他角色之間的成長和變化。

**Comment by B1nn9** [comment]
我推薦《在異世界獲得超強能力的我，在現實世界照樣無敵》。這本書講述了主角在祖父的房間裡發現了通往異世界的大門，並在那裡獲得了賢者的遺產和強大的能力，於是主角開始往返兩個世界，利用自己的技能和物品改變自己的人生，並與各種有趣的人物和怪物相遇。他不僅在異世界展現了無敵的實力，也在現實世界獲得了財富、名聲和愛情。故事的主要看點是主角的成長和冒險，以及他與異世界和現實世界的角色之間的感情和友情，讓讀者可以欣賞到主角如何運用自己的勇氣和能力，來克服兩個世界的挑戰和危機，也可以感受到他與其他角色之間的溫馨與幽默。

**Comment by B1nn9** [comment]
我推薦《不時輕聲地與俄語遮羞的鄰座艾莉同學》。這本書講述了男主角和他的鄰座同學之間的戀愛喜劇。女主角是一位日俄混血的美少女，學業成績優秀，運動萬能，任職學生會會計，被稱為「孤高公主」。但她其實有個秘密，就是在害羞或對男主角生氣時，會在他面前小聲地以俄語說出自己的真心話，卻不知道男主角其實也會俄語，因此他能聽懂她的一切。在這樣的情況下，兩人的關係逐漸發展，並產生了許多有趣的情節。故事的主要看點是女主角的俄語遮羞和男主角的反應，以及他們之間的甜蜜互動和感情進展。讀者可以欣賞到女主角的可愛和男主角的包容，也可以感受到他們的成長和變化。另外，這本書的動畫由動畫工房製作，預計於2024年播出。

**Comment by thz0000** [comment]
輕小說的經典我第一時間會想到<妖怪公館的新房客>
雖然這好像是我國中時的事了，但還是覺得很值得推薦
我覺得這本書友誼和有趣共存，當然也有賺熱淚的部分，而且每個人物個性鮮明，會讓人想重刷
以下是故事簡介：

剛升上高中的高中一年級人類新生封平瀾，本來已經進入宿舍準備展開美好的校園生活，卻因為某些流程出錯，陷入了沒有宿舍的窘境。此時宿舍管理員介紹了一間房屋，封平瀾就這樣誤打誤撞地闖入傳說中的「鬼屋」，更不小心解開了以奎薩爾為首的妖魔們的封印，從此和六隻妖魔展開了混亂的校園及同居生活。

**Comment by chiuwangwang** [comment]
我推薦一本小說，雖然我覺得不是輕小說，<鑄劍師>，我覺得非常好看，雖然都是短篇故事，但每篇卻都互相串連。主要講的是春秋戰國的故事，如果像我喜歡中國歷史的話，我非常推薦這本，與歷史相扣，又有許多野史的痕跡，我十分推薦這本。
